client/components/examples/activity/NewActivity.vue

```diff
@@ -161,8 +166,18 @@ const ExampleGetter = namespace('examples/index', Getter)
 export default class NewActivity extends Vue {
   @ExampleGetter labels
   @ExampleGetter organizers
+  @ExampleGetter cities
```

Comment:
God I wished I could understand what black magic happens with `const ExampleGetter = namespace('examples/index', Getter)`.
I couldn't find what this does, and how to properly use this.

Reply:
You can have a look at https://github.com/ktsn/vuex-class 😸